_posts/2022-04-19-small-development-grant.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title:  "NumFOCUS small development grant: adding complex number support to JuMP"
+date:   2022-04-19
+categories: [announcements]
+---
+
+The [JuMP Steering Committee](/pages/governance/#steering-committee) is pleased
+to announce that we have received a
+[small development grant](https://numfocus.org/programs/small-development-grants)
+from [NumFOCUS](https://numfocus.org) to add complex number support to JuMP.
+
+Currently, JuMP and MathOptInterface are limited to formulating optimization
+problems with real-valued decision variables. However, complex numbers appear in
+a variety of industrial optimization problems. The most important of these is
+the AC optimal power flow problem (AC-OPF), which is used world-wide to optimize
+the operations of electrical power grids. Currently, users in JuMP are forced to
+decompose their problem into the real and complex parts. This leads to a
+duplication in which twice as many variables are needed, reducing readability
```

Comment:
wont the variables still be duplicated for practical solvers?

Reply:
Yes, but the problem is mainly in the readability. If it spurs solvers to support complex values natively, that's also a good outcome.

Reply:
Yes, most solvers do not support complex numbers yet